### PR TITLE
Get started docs

### PR DESCRIPTION
**Pull request by nadvolod**
# One-line summary

Cleanup and update of docs

## Description
Restructuring the menu, updating docs

## Types of Changes
- Documentation / non-code

## Review
_List of tasks the reviewer must do to review the PR_
- [x] Documentation

## Deployment Notes
These should highlight any db migrations, feature toggles, etc.

### CODE REVIEW

File: website/docs/getting-started/intro.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+✅ Automatic session cleanup and status reporting
+
+✅ No magic strings
+
+✅ Less docs - let the IDE guide your automation
+
+## Less code and maintenance
+
+```java title="withoutSauceBindings.java"
```
Comment (diemol) [inline website/docs/getting-started/intro.mdx:24]: Is there a way to display these two samples next to each other?

File: website/docs/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+There are a number of features we will be looking to add.
+If there is something you'd like to see that isn't on this list, please
+[open a ticket](https://github.com/saucelabs/sauce_bindings/issues/new).
+
+This is the current plan for major features in future versions. Point releases may be created
+to address language-specific bugs.
+
+## 2.0
+
+- UPDATE ME
```
Comment (diemol) [inline website/docs/roadmap.md]: Can we avoid this "UPDATE ME"? Perhaps leave the bullet points shown below as the roadmap.

File: website/docs/getting-started/intro.mdx (hunk 1 above)
Comment (nadvolod) [reply website/docs/getting-started/intro.mdx:24]: @diemol i thought of the same thing. After trying 2 options, I couldn't get it to easily work. I thought it's better to add all the content on this page and then come back and improve it. Are you okay with that?

Comment (nadvolod) [reply website/docs/getting-started/intro.mdx:24]: @diemol I created this to update in the future https://github.com/saucelabs/sauce_bindings/issues/323

**Pull request merged**